### PR TITLE
organize multiple keeper registries and generate gethwrappers

### PR DESCRIPTION
**Pull request by infiloop2**
List of changes

- Keep v1.1 registry inside v0.7 folder. rename to KeeperRegistry1_1 to avoid collisions and make it clear it's an old version
- in v0.8, move dev contracts to main folder. Remove Dev from file names
- Cleanup KeeperRegistry.test.ts and KeeperRegistrar.test.ts
- Create two versions of wrapper: keeper_registry_wrapper1_1 and keeper_registry_wrapper1_2. Delete keeper_registry_wrapper
- Replace all occurences of keeper_registry_wrapper with keeper_registry_wrapper1_1

### CODE REVIEW

File: contracts/test/v0.7/KeeperRegistry1_1.test.ts [hunk 1]
```diff
@@ -4,11 +4,11 @@ import { evmRevert } from '../test-helpers/matchers'
 import { getUsers, Personas } from '../test-helpers/setup'
 import { BigNumber, Signer, BigNumberish } from 'ethers'
 import { LinkToken__factory as LinkTokenFactory } from '../../typechain/factories/LinkToken__factory'
-import { KeeperRegistry__factory as KeeperRegistryFactory } from '../../typechain/factories/KeeperRegistry__factory'
+import { KeeperRegistry11__factory as KeeperRegistryFactory } from '../../typechain/factories/KeeperRegistry11__factory'
 import { MockV3Aggregator__factory as MockV3AggregatorFactory } from '../../typechain/factories/MockV3Aggregator__factory'
 import { UpkeepMock__factory as UpkeepMockFactory } from '../../typechain/factories/UpkeepMock__factory'
 import { UpkeepReverter__factory as UpkeepReverterFactory } from '../../typechain/factories/UpkeepReverter__factory'
-import { KeeperRegistry } from '../../typechain/KeeperRegistry'
+import { KeeperRegistry11 as KeeperRegistry } from '../../typechain/KeeperRegistry11'
```
[the remark this answers is not in the document]
Comment (infiloop2) [reply contracts/test/v0.7/KeeperRegistry1_1.test.ts:11]: this is autogenerated by hardhat, couldn't find a way around it

[the remark this answers is not in the document]
Comment (cmalec) [reply contracts/test/v0.7/KeeperRegistry1_1.test.ts:11]: ah gotcha bummer ok

**Review by cmalec — APPROVED** [review]

File: contracts/test/v0.7/UpkeepRegistrationRequests.test.ts [hunk 2]
```diff
@@ -29,7 +29,8 @@ before(async () => {
   mockV3AggregatorFactory = (await ethers.getContractFactory(
     'src/v0.7/tests/MockV3Aggregator.sol:MockV3Aggregator',
   )) as unknown as MockV3AggregatorFactory
-  keeperRegistryFactory = await ethers.getContractFactory('KeeperRegistry')
+  // @ts-ignore bug in autogen file
```
Comment (RyanRHall) [inline contracts/test/v0.7/UpkeepRegistrationRequests.test.ts:32]: does using the full import path resolve the TS bug? example on line 30.

**Review by RyanRHall — APPROVED** [review]

File: contracts/test/v0.7/UpkeepRegistrationRequests.test.ts (hunk 2 above)
Comment (infiloop2) [reply contracts/test/v0.7/UpkeepRegistrationRequests.test.ts:32]: unfortunately no, this is a hardhat issue with naming. If I put 'KeeperRegistry11' then typechecker is happy but it fails while running

Comment (infiloop2) [reply contracts/test/v0.7/UpkeepRegistrationRequests.test.ts:32]: Essentially solc is able to handle the '_', but hardhat can't.

**Pull request merged**